atlp-rwanda/bravo-bn-fe
# Chore#182201674 configure storybook for components development

#### What does this PR do?
 - configure storybook for component development
#### Description of Task to be completed?
 - initialize storybook
 - create an initial example story
#### How should this be manually tested?
 - clone this repository
 - checkout to this branch `ch-configure-storybook-182201674`
 - run `npm install` to install dependencies
 - run `npm run storybook` to start storybook
 - open it in the browser on `http://localhost:6007/` or any other specified port it might be running on 
to see the storybook interface
#### What are the relevant pivotal tracker stories?
 - https://www.pivotaltracker.com/n/projects/2569105/stories/182201674
#### Screenshots(If appropriate)
> <img width="400" alt="Screen Shot 2022-07-26 at 00 17 42" src="https://user-images.githubusercontent.com/73250459/180883403-1fcf33a5-5dd0-4941-9cc1-be33e3b0e0a1.png">
